local-cli/generator-swift/templates/tests/Tests.swift

```diff
@@ -0,0 +1,62 @@
+/**
+* Copyright (c) 2015-present, Facebook, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the BSD-style license found in the
+* LICENSE file in the root directory of this source tree. An additional grant
+* of patent rights can be found in the PATENTS file in the same directory.
+*/
+
+import UIKit
+import XCTest
+
+let TIMEOUT_SECONDS: Double = 240
+let TEXT_TO_LOOK_FOR: String = "Welcome to React Native!"
+
+class <%= name %>Tests: XCTestCase {
+
+	override func setUp() {
+		super.setUp()
+	}
+
+	override func tearDown() {
+		super.tearDown()
+	}
+
+	func testRendersWelcomeScreen() {
+		let vc: UIViewController? = UIApplication.sharedApplication().delegate?.window?!.rootViewController
+		let date: NSDate = NSDate(timeIntervalSinceNow: TIMEOUT_SECONDS)
+		var foundElement: Bool = false
+		var redboxError: String? = nil
+
+		RCTSetLogFunction({ (level, fileName, lineNumber, message) -> () in
+			if level.rawValue >= RCTLogLevel.Error.rawValue {
+				redboxError = message
+			}
+		})
+
+		while date.timeIntervalSinceNow > 0 && !foundElement && redboxError == nil {
+			NSRunLoop.mainRunLoop().runMode(NSDefaultRunLoopMode, beforeDate: NSDate(timeIntervalSinceNow: 0.1))
+			NSRunLoop.mainRunLoop().runMode(NSRunLoopCommonModes, beforeDate: NSDate(timeIntervalSinceNow: 0.1))
+
+			foundElement = findSubview(vc!.view!, matchingTest: { (view) -> Bool in
+				return view.accessibilityLabel != nil && view.accessibilityLabel.isEqual(TEXT_TO_LOOK_FOR)
```

Comment:
Minor, but Xcode 7 GM wants to correct this to

```swift
return view.accessibilityLabel != nil && view.accessibilityLabel!.isEqual(TEXT_TO_LOOK_FOR)
```
I'm no expert, but I think it has to do with upgrading to Swift 2.0